config/crd/bases/operator.knative.dev_knativeservings.yaml

```diff
@@ -2299,6 +2299,8 @@ spec:
                         type: boolean
                       service-type:
                         type: string
+                      service-loadBalancerIP:
```

Comment:
`loadBalancerIP` is same style with k8s's but camel case and kebab case mixed.
@houshengbo Do you have any suggestion for the config name - like `service-loadBalancerIP`, `service-load-balancer-ip` or somethinge else?

Reply:
I'm happy to change this to `service-load-balancer-ip` if that's better. Let me know.

Reply:
I am prefer to `service-load-balancer-ip`. WDYT @houshengbo ?

Reply:
Done